packages/matter-node.js/src/matter/mdns/MdnsScanner.ts

```diff
@@ -38,15 +38,15 @@ export class MdnsScanner implements Scanner {
         this.periodicTimer = Time.getPeriodicTimer(60 * 1000 /* 1 mn */, () => this.expire()).start();
     }
 
-    async findDevice({ operationalId }: Fabric, nodeId: NodeId): Promise<MatterServer | undefined> {
+    async findDevice({ operationalId }: Fabric, nodeId: NodeId, timeoutSeconds = 5): Promise<MatterServer | undefined> {
         const operationalIdString = operationalId.toHex().toUpperCase();
         const deviceMatterQname = getDeviceMatterQname(operationalIdString, nodeId.toString());
 
         const record = this.matterDeviceRecords.get(deviceMatterQname);
         if (record !== undefined) return { ip: record.ip, port: record.port };
 
         const { promise, resolver } = await getPromiseResolver<MatterServer | undefined>();
-        const timer = Time.getTimer(5 * 1000 /* 5 s*/, () => {
+        const timer = Time.getTimer(timeoutSeconds * 1000 /* timeOut seconds */, () => {
```

Comment:
Not sure what value the `/* timeOut seconds */` comment is adding.

Reply:
Also, if the underlying resolution of the timer is milliseconds, would it make sense to simply expose `timeoutMs` instead of `timeoutSeconds` and multiplying out?

Reply:
I removed the unneeded comment. The matter specs mostly use seconds so lets use seconds

Reply:
Yeah, align with spec is always good justification.